src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -222,7 +222,7 @@ export class MeetingManager implements AudioVideoObserver {
     meetingManagerConfig,
   }: MeetingJoinData) {
     if (meetingManagerConfig) {
-      this.meetingManagerConfig = meetingManagerConfig;
+      this.meetingManagerConfig = {...this.meetingManagerConfig, ...meetingManagerConfig};
```

Comment:
This is necessary, cause if MeetingProvider creates a MeetingManager with few props and later at join time if builders provide a custom meetingManagerConfig we override the existing one, rather override the attributes that are passed and not all.

Reply:
Just want to confirm. For example:
```js
this.meetingManagerConfig = {
    a: 0,
    b: 0,
}

meetingManagerConfig = {
    b: 1,
    c: 1
};

this.meetingManagerConfig = { ...this.meetingManagerConfig, ...meetingManagerConfig };

console.log(this.meetingManagerConfig);
// {
//   "a": 0, // case 1
//   "b": 1, // case 2
//   "c": 1, // case 3
// } 
```
So what we want is to:
* case 1: keep the existing properties which `meetingManagerConfig` doesn't have
* case 2: override the existing properties which `meetingManagerConfig` does have
* case 3: add the new properties which`meetingManagerConfig` dose have

Reply:
Yes, I think that is better over the existing direct replace approach, what do you think? I was hesitant on doing so because its easier to just say that the MeetingManagerConfig from join overrides the existing MeetingProvider props but then why would we need the MeetingProvider props in first place.

Reply:
Ok, so I checked few last PRs and we said if you are passing the props through it will just override the MeetingManagerConfig:
Source: Check "meetingManager.join()` under documentation [here](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/sdk-providers-meetingmanager--page).
```
// Set the meeting config. This will override the props from MeetingProvider.
  meetingManagerConfig: MeetingManagerConfig;
```

Lets follow that and just update the documentation should be enough. I will re-work this.

Reply:
I think it's a breaking change. Because it is not guaranteed to override the whole `meetingManagerConfig` anymore, unless the builders overrides all the properties.

Reply:
As discussed will not be doing so.